device-types/APC/AP8841.yaml

```diff
@@ -0,0 +1,184 @@
+---
+manufacturer: APC
+model: AP8841
+slug: apc-ap8841
+part_number: AP8841
+u_height: 0
+is_full_depth: false
+comments: Rack PDU 2G, metered, 0U, 30A, 200/208V, (36) C13 & (6) C19. (2) 20A 2-pole hydraulic-magnetic circuit breakers

```

Comment:
Please add the `airflow` data point if you have the information

Reply:
This particular unit has no fan for any airflow.

Reply:
Then the data point should be `airflow: passive`